src/lib/DepositFormApp.js

```diff
@@ -112,4 +106,6 @@ DepositFormApp.defaultProps = {
   draftsService: null,
   filesService: null,
   recordSerializer: null,
+  files: undefined,
```

Comment:
this can be described more accurately in the Proptypes e.g `Proptypes.shape(...)`

Reply:
maybe is better to fallback to `undefined` always instead of null for all the missing props....Do we need to define value for optional properties actually? Can you test if removing them works? I mean the `null`/`undefined`...

Reply:
Yes it is a rule that every propType that is not `required` should have a defaultProp, as you see i usually default to undefined so that the behaviour does not change unexpectedly.

as for setting the shape i am not sure what that could be in this case 😅